lib/resources/interface.rb

```diff
@@ -87,28 +113,53 @@ def interface_info(iface)
         speed = convert_to_i(speed)
       end
 
+      family_addresses = addresses(iface)
       {
         name: iface,
         up: state,
         speed: speed,
+        ipv4_addresses: family_addresses['inet'],
+        ipv6_addresses: family_addresses['inet6'],
       }
     end
+
+    private
+
+    def addresses(iface)
+      cmd = inspec.command("echo -n 'inet6 ' && /sbin/ip -br -6 address show dev #{iface} && echo '' && " \
```

Comment:
Looks like this needs to either be need be ported to MacOS (no /sbin/ip), or add a guard clause indicating it is not supported (along with a `## Limitations` section in the docs).

@miah do you have any portability concerns?

Reply:
I wasn't concerned with macOS as this class is already Linux specific. e.g. `/sys/class/net` doesn't exist on macOS either and it's called LinuxInterface.

Reply:
Yes, good call.